kyo-combinators/shared/src/main/scala/kyo/Constructors.scala

```diff
@@ -111,12 +107,8 @@ extension (kyoObject: Kyo.type)
       * @return
       *   A new sequence with elements collected using the function
       */
-    def foreachPar[A, S, A1](
-        sequence: => Seq[A] < S
-    )(
-        useElement: A => A1 < Async
-    )(using Flat[A1], Frame): Seq[A1] < (S & Async) =
-        sequence.map(seq => Async.parallel(seq.map(useElement)))
+    def foreachPar[A, S, A1](sequence: Seq[A])(useElement: A => A1 < Async)(using Flat[A1], Frame): Seq[A1] < Async =
```

Comment:
Can this support Ctx?

Reply:
Added! It was also missing test coverage